### ISSUE TITLE
Strange stderr errors

### ISSUE THREAD

**Issue by Vagabond** [benoitc/inet_ext#4]
When using inet_ext via erlang-nat we sometimes see messages on stderr like this:

```
erl_child_setup: failed with error 32 on line 253
```
```
cut: write error: Broken pipe
```

They don't appear consistently, and I'm not sure why.

**Comment by benoitc** [comment]
no trace? On which OS do you get it? Seems like it get the error from there. 

https://github.com/benoitc/inet_ext/blob/master/src/inet_ext.erl#L60

**Comment by Vagabond** [comment]
Yes, exactly. It happens on linux and if I run the shell command manually it never throws a broken pipe.

**Comment by benoitc** [comment]
it may be the cut command is not finished before cut is ending , do you always reproduce it on the same interface?

**Comment by benoitc** [comment]
bump, i am thinking for a simple patch to eventually retry if it happens. Maybe it's an issue with the number of fds available .

**Comment by benoitc** [comment]
@Vagabond i wasn't able to reproduce the issue yet. Can you five me more context. Also is this true linux or WSL ?

**Comment by benoitc** [comment]
@Vagabond can you try to set the vm args [+spp](http://erlang.org/doc/man/erl.html#+spp) to true and let me know if it solves the issue for you?

**Comment by Vagabond** [comment]
This is true linux, I will see if I can find out how to set +spp, I see it running unit tests, so it's a bit tricky to figure out how to enable that flag under rebar3.

**Comment by benoitc** [comment]
@Vagabond is there any way I can run the unit tests myself? I didn't reproduced it yet.

**Comment by Vagabond** [comment]
Yeah, if you run the erlang-libp2p test suite with `make test` you'll see them.
